### PR TITLE
Improve timeout timestamp log when query in QueryNode

### PR DESCRIPTION
**Pull request by bigsheeper**
Signed-off-by: bigsheeper <yihao.dai@zilliz.com>

issues: #14689

### CODE REVIEW

File: internal/querynode/query_collection.go [hunk 1]
```diff
@@ -318,16 +318,16 @@ func (q *queryCollection) setServiceableTime(t Timestamp) {
...
-	//	zap.Any("msgID", msg.ID()),
-	//	zap.Any("TimeoutTs", msg.TimeoutTs()),
-	//	zap.Any("curTime", curTime),
-	//	zap.Any("timeoutTsPhysical", timeoutTsPhysical),
-	//	zap.Any("curTimePhysical", curTimePhysical),
-	//)
+	curTimePhysical, _ := tsoutil.ParseTS(curTime)
+	timeoutTsPhysical, _ := tsoutil.ParseTS(msg.TimeoutTs())
+	log.Debug("check if query timeout",
+		zap.Any("collectionID", q.collectionID),
```
Comment (xige-16) [inline internal/querynode/query_collection.go:324]: useing zap.any is not recommended

Comment (bigsheeper) [reply internal/querynode/query_collection.go]: fixed

**Pull request merged**